android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -886,10 +886,12 @@ private static void copyAssets(Context context) {
 
       // Copy default keyboard font
       copyAsset(context, KMDefault_KeyboardFont, "", true);
+      // Includes something needed up to Chrome 61, which has full ES5 support.
+      // Thus, this one isn't legacy-only.
+      copyAsset(context, KMFilename_JSPolyfill2, "", true);
```

Comment:
Addresses a side detail, though that side detail's documentation is within the same base issue.  Got lumped together there, so "why not here?"